lib/elixir/lib/calendar/time.ex

```diff
@@ -355,13 +355,21 @@ defmodule Time do
 
   ## Examples
 
+      iex> Time.from_erl({23, 30, 15})
```

Comment:
The docs were lacking the basic `from_erl` case, `from_erl!` has it, so I added one here too.